src/cli/commands.ts

```diff
@@ -52,6 +52,10 @@ export default {
           boolean: false,
           description: 'Url prefix for registry server',
           default: ''
+        },
+        components: {
+          array: true,
+          description: 'Filter only the components that you want to be included'
```

Comment:
How about: `List of component names that you want to be packaged and exposed on dev registry`

Reply:
done